### ISSUE TITLE
Change default Explorer config to expand the tree to the current open file (even if a file is inside a hidden or "gitignored" folder)

### ISSUE THREAD

**Issue by AIGeneratedUsername** [AstroNvim/AstroNvim#2428]
### TL;DR

I am new to nvim, but I think this behaviour should not break for the existing user and should have no side effects. 

### AstroNvim default behavior 

1. You are inside `my_project_folder/`.
2. You open a file (e.g. `app.py`).
2. Via LSP you navigate to a dependency file (e.g. `dependency.py`) that is inside `my_project_folder/.venv/`.
3. You press SPACE + E.
4. Explorer opens and does not expand the tree:
   ```
   my_project_folder/
       app.py
   ```

### LunarVim default behavior 

1. You are inside `my_project_folder/`.
2. You open a file (e.g. `app.py`).
2. Via LSP you navigate to a dependency file (e.g. `dependency.py`) that is inside `my_project_folder/.venv/`.
3. You press SPACE + E.
4. Explorer opens and expands the tree:
   ```
   my_project_folder/
       .venv/
           third_party_package/
               dependency.py
       app.py
       .gitignore
   ```

### What I want

I want LunarVim default behavior:  
When I have an open deeply nested file, I can press SPACE + E and open/view files around.

**Comment by mehalter** [comment]
By default we don't show hidden files as a normal default behavior of file explorers. It would be nice if we could have neo-tree not show hidden files but then selectively enable it if we are currently in a hidden file. This would be a good feature request upstream.

If you want this behavior now, you can definitely enable hidden files in your config to make it work how you want, but this isn't a default that we want to add into AstroNvim. Thanks for the recommendation!

**Comment by mehalter** [comment]
To enable visibility of hidden files you just want to add this to your plugins:

```lua
{
  "nvim-neo-tree/neo-tree.nvim",
  opts = {
    filesystem = {
      filtered_items = {
        visible = true,
      },
    },
  },
}
```

**Comment by mehalter** [comment]
I have opened up a feature request upstream and we will see if it gets implemented! It would be a nice addition :)
